bin/dev/plot_slice_surf.py

```diff
@@ -1,4 +1,30 @@
 # -*- encoding: utf-8 -*-
+# Tools for building full brain network models from standard structural MR scans.
+# Web-UI helpful to run brain-simulations. To use it, you also need do download
+# TheVirtualBrain-Scientific Package (for simulators). See content of the
+# documentation-folder for more details. See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2017, Baycrest Centre for Geriatric Care ("Baycrest") and others
```

Comment:
This is not the correct Copyright text. If we decide to include it in all scripts, should be the text from https://github.com/the-virtual-brain/tvb-recon/blob/master/LICENSE.
But, for the moment, I see the other scripts do not have a copyright at top.

Reply:
ok i will used that

Reply:
NO, rather leave it as it was (no copyright text in scripts). We might add it only later.

Reply:
Ok Thanks, Right away